pep-0XXX-environment-markers.rst

```diff
@@ -0,0 +1,174 @@
+PEP: 0XXX
+Title: Environment Markers
+Version: $Revision$
+Last-Modified: $Date$
+Author: James Polley <jp@jamezpolley.com>
```

Comment:
Done.

A few lines further down on the Replaces: line, I wasn't sure how to say that this is only a partial replacement for PEP-0345; is there a better way to do that?